### DISCUSSION TITLE
Min and Max ignored for time

### DISCUSSION THREAD

**Discussion by beard7** [capawesome-team/capacitor-plugins#219] [Q&A]
I can't get the min and max range to apply for te time section of a datetime picker.

These are the options I'm using:
````
options = {
  cancelButtonText: 'Cancel',
  doneButtonText: 'Ok',
  mode: "datetime",
  format: "yyyy-MM-dd'T'HH:mm",
  value: "2024-07-03T06:28",
  min: "2024-07-03T03:00",
  max: "2024-07-03T09:00",
  theme: 'dark',
  locale: 'en-GB',
  androidTimePickerMode: 'clock',
  androidDatePickerMode: 'calendar',
};
````
Any suggestions?

Thanks

**Comment by robingenz** [comment]
Please take a look at the [documentation](https://capawesome.io/plugins/datetime-picker/#presentoptions):

> The format of this value must match the value of the format parameter. 

Your values does not match the value of the format parameter.

**Comment by beard7** [comment]
Hi.

I thought I had followed the documentation. In what way do the values not match the specified format?

**Reply by robingenz** [reply]
You're right, that was my mistake. It seems to be correct. Please create a bug report so I have all the information I need to debug the problem.
